tests/tests_pytorch/run_standalone_tests.sh

```diff
@@ -51,7 +51,7 @@ function show_batched_output {
   if [ -f standalone_test_output.txt ]; then  # if exists
     cat standalone_test_output.txt
     # heuristic: stop if there's mentions of errors. this can prevent false negatives when only some of the ranks fail
-    if grep -iE 'error|exception|traceback|failed' standalone_test_output.txt | grep -qvE 'on_exception|xfailed'; then
+    if grep -iE 'error|exception|traceback|failed' standalone_test_output.txt | grep -qvE 'on_exception|xfailed|test_find_unused_parameters_exception'; then
```

Comment:
I would rename the test to `test_find_unused_parameters_raises` which is what we for other similar tests